app/experimenter/nimbus-ui/src/lib/visualization/mocks.tsx

```diff
@@ -27,6 +27,12 @@ export const MOCK_METADATA = {
         "This is a metric description. It's made by a data scientist at the creation time of the metric [I'm a link](https://www.example.com)",
       friendly_name: "Feature D Friendly Name",
     },
+    search_count: {
+      bigger_is_better: true,
+      description:
+        "This is a metric description for search. It's made by a data scientist at the creation time of the metric [Link](https://www.example.com)",
```

Comment:
What will this be linking to out of curiosity? This may be out of line but should this be like, `This is a [metric description for search](https://www.example.com)]. It's made by a data scientist at the creation time of the metric.`

Reply:
Oh, well the descriptions are coming in from Jetstream, they links would be more details for search, for example. I suppose I could update it. I just added the link to show it's possible to have a link and how it would work.